tests/baselines/reference/moduleWithNoValuesAsType.errors.txt

```diff
@@ -1,20 +1,20 @@
-tests/cases/compiler/moduleWithNoValuesAsType.ts(2,8): error TS2304: Cannot find name 'A'.
-tests/cases/compiler/moduleWithNoValuesAsType.ts(7,8): error TS2304: Cannot find name 'B'.
-tests/cases/compiler/moduleWithNoValuesAsType.ts(15,8): error TS2304: Cannot find name 'C'.
+tests/cases/compiler/moduleWithNoValuesAsType.ts(2,8): error TS2708: Cannot use namespace 'A' as a value here because it has no value export.
+tests/cases/compiler/moduleWithNoValuesAsType.ts(7,8): error TS2708: Cannot use namespace 'B' as a value here because it has no value export.
+tests/cases/compiler/moduleWithNoValuesAsType.ts(15,8): error TS2708: Cannot use namespace 'C' as a value here because it has no value export.
 
 
 ==== tests/cases/compiler/moduleWithNoValuesAsType.ts (3 errors) ====
     module A { }
     var a: A; // error
            ~
-!!! error TS2304: Cannot find name 'A'.
+!!! error TS2708: Cannot use namespace 'A' as a value here because it has no value export.

```

Comment:
The more specific error here is now wrong instead of being vague.

I think I like 'wrong' over 'vague' though.